libfwupd/README.md

```diff
@@ -3,21 +3,29 @@
 ## Planned API/ABI changes for next release
 
 * Typedef `FwupdFeatureFlags` to `guint64` so it's the same size on all platforms
-* Remove the `soup-session` fallback property in `FwupdClient`.
-* Remove fwupd_device_set_vendor_id() and fwupd_device_get_vendor_id()
-* Remove the deprecated flags like `FWUPD_DEVICE_FLAG_MD_SET_ICON`
-* Remove `fwupd_release_get_uri()` and `fwupd_release_set_uri()`
 * Rename `fwupd_client_install_release2_async()` to `fwupd_client_install_release_async()`
-* Remove fwupd_device_set_protocol() and fwupd_device_get_protocol()
 * Remove deprecated install flag `FWUPD_INSTALL_FLAG_IGNORE_POWER`
 * Rename `fwupd_remote_set_checksum()` to `fwupd_remote_set_checksum_sig()`
-* Remove the deprecated flag `FWUPD_CLIENT_DOWNLOAD_FLAG_ONLY_IPFS`
 * Rename `fwupd_client_refresh_remote2_async()` to `fwupd_client_refresh_remote_async()`
 * Remove the deprecated method `fwupd_remote_get_enabled()`
 * Remove the deprecated method `fwupd_remote_get_approval_required()`
 * Remove the deprecated method `fwupd_remote_get_automatic_reports()`
 * Remove the deprecated method `fwupd_remote_get_automatic_security_reports()`
```

Comment:
let's make sure we merge https://github.com/fwupd/fwupd/pull/6530 before this branch and then fix those things there too